core/src/main/kotlin/team/backend/support/Compression.kt

```diff
@@ -0,0 +1,50 @@
+package team.backend.support
+
+import java.io.ByteArrayOutputStream
+import java.util.zip.Deflater
+import java.util.zip.Inflater
+
+/**
+ * Compress a string using ZLIB.
+ *
+ * @return an UTF-8 encoded byte array.
+ */
+fun String.zlibCompress(): ByteArray {
```

Comment:
I think it would be better to have the description of the action that this method performs at the beginning. or it would be okay to exclude 'zlib' if there is no other algorithm than zlib.

e.g. compressZlib. compress, decompressZlib, decompress